R/g.plot5.R

```diff
@@ -91,60 +100,79 @@ g.plot5 = function(metadatadir = c(), dofirstpage = FALSE, viewingwindow = 1, f0
           }
           if (length(della) > 0) summarysleep_tmp = summarysleep_tmp[-della,]
         }
-        # do not include days with no meaningful data
         threshold_hrs_of_data_per_day = 0.5
```

Comment:
If I understand well, this requires half an hour with valid data for the day (full day, including day and spt), shouldn't this be consistent with `includedaycrit`?

Reply:
Filtering based on `includedaycrit` happens further down in the code and is only done if `visualreport_without_invalid` is set to TRUE. However, the line here ensures that days are only plotted if they have at least half an hour of data regardless of whether they are valid or invalid. I remember inserting this as sometimes the first or last day of data only has one epoch of data.